packages/ckeditor5-core/src/plugincollection.js

```diff
@@ -423,6 +427,83 @@ export default class PluginCollection {
 				return promise.then( plugin[ method ].bind( plugin ) );
 			}, Promise.resolve() );
 		}
+
+		/**
+		 * Replaces plugin constructors with the specified set of plugins. A useful option for replacing built-in plugins while creating
```

Comment:
The "useful" part makes sense in `init()`, not here. That's where you might be looking for it.